src/main/java/com/craftaro/epicfarming/database/migrations/_1_InitialMigration.java

```diff
@@ -12,7 +12,7 @@
 
 public class _1_InitialMigration extends DataMigration {
     public _1_InitialMigration() {
-        super(1);
+        super(2);
```

Comment:
I think you changed this to a `2` by accident?

Reply:
my bad, this has been corrected, thank you for catching this!